java/src/jmri/jmrix/openlcb/OlcbConfigurationManager.java

```diff
@@ -243,6 +243,12 @@ public boolean provides(Class<?> type) {
         if (type.equals(jmri.LightManager.class)) {
             return true;
         }
+        if (type.equals(jmri.GlobalProgrammerManager.class)) {
+            return true;
+        }
+        if (type.equals(jmri.AddressedProgrammerManager.class)) {
+            return true;
+        }
```

Comment:
Not sure, but it seems to me that these if statements should have:
`return !adapterMemo.getDisabled();`

If `adapterMemo.getDisabled()` returns true, the method `getProgrammerManager()` returns null.
But maybe there is a lot of this practice in JMRI.

@bobjacobsen 
What's your thoughts about this?

Reply:
My fault. That check is done in the beginning of the method so no need to do it here.